streams/src/test/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImplTest.java

```diff
@@ -153,23 +153,23 @@ public void shouldNotHaveInvalidStoreNameOnWindowedAggregate() {
             .aggregate(MockInitializer.STRING_INIT, MockAggregator.TOSTRING_ADDER, Materialized.as(INVALID_STORE_NAME));
     }
 
-    private void doAggregateSessionWindows(final Map<Windowed<String>, Integer> results) {
+    private void doAggregateSessionWindows(final Map<Windowed<String>, KeyValue<Integer, Long>> results) {
```

Comment:
Updating this test to also check the timestamp -- also add out-of-order record to the test case.